public/locales/pt/createPet.json

```diff
@@ -0,0 +1,85 @@
+{
+  "createPetTitle": "Crie um novo mascote",
+  "editTo": "Alterar {{ name }}",
+  "backPets": "Volta do perfil",
+  "placeholderImages": "Adicionar imagem do mascote",
+  "addFile": "Escolher uma imagen",
+  "assignUserTransit": "Atribuir usuário de trânsito",
+  "assignUser": "Atribuir adotante",
+  "labelWhereFoundPet": "¿Onde foi encontrada?",
```

Comment:
In portuguese we don't have "¿" in questions like in spanish. I recommend to remove